tests/app/TestKernel.php

```diff
@@ -61,4 +61,22 @@ public function registerContainerConfiguration(LoaderInterface $loader)
             ]);
         });
     }
+
+    public function getCacheDir()
+    {
+        if (method_exists($this, 'getProjectDir')) {
+            return $this->getProjectDir() . '/tests/app/cache/' . $this->getEnvironment();
+        }
+
+        return parent::getCacheDir();
+    }
+
+    public function getLogDir()
+    {
+        if (method_exists($this, 'getProjectDir')) {
+            return $this->getProjectDir() . '/tests/app/cache/' . $this->getEnvironment();
+        }
+
+        return parent::getLogDir();
+    }
```

Comment:
I assume this fixes some deprecation? Can you explain a bit more?

Reply:
this is not deprecation fix, but just fix. With latest vendors after running test suite I got cache and logs dir not in their places. After some investigation I found that there were changes Symfony HttpKernel and we need tune this place